samples/file-encryptor/enclave/keys.cpp

```diff
@@ -368,20 +359,25 @@ int ecall_dispatcher::parse_encryption_header(
         goto exit;
     }
 
+    memcpy(salt, header->salt, SALT_SIZE_IN_BYTES);
```

Comment:
Same here. Can we check if the header is null in this function as well?

Reply:
added null check